### ISSUE TITLE
[11.0] open form in edit mode

### ISSUE THREAD

**Issue by yousef-soliman** [odoo/odoo#22353]
Impacted versions:
11.0
Steps to reproduce:
make python function that return Action
```python
return {
            'res_model':'res.company',
            'view_mode':'form',
            'res_id':self.env.user.company_id.id,
            'type':'ir.actions.act_window',
            'view_type':'form',
            'view_id':self.env.ref('cbpo_client_dashboard.client_dashboard_form').id, 
            'target':'current',
            'flags': {'initial_mode': 'edit'},   
        }
```
Current behavior:
not open form in edit mode 
Expected behavior:
open form in edit mode

Video/Screenshot link (optional):

**Comment by sswapnesh** [comment]
@yousef-soliman 
I can confirm this Issue and can reproduce it on [Runbot](http://295137-11-0-66a95d.runbot13.odoo.com/web?debug#id=1&view_type=form&model=mail.mass_mailing.campaign&menu_id=219&action=258).

![](https://image.prntscr.com/image/gCYiyffWQgqtEA7MyvMyUw.png)

Here is the code: [https://github.com/odoo/odoo/blob/11.0/addons/mass_mailing/models/mass_mailing.py#L530](https://github.com/odoo/odoo/blob/11.0/addons/mass_mailing/models/mass_mailing.py#L530)

CC @Yenthe666

**Comment by mge-odoo** [comment]
Hi @yousef-soliman 

Thanks for reporting the issue.

The flag `initial_view` was in fact working by chance in 10.0 because the view attribute has the same name.

We have tried to clean up the options (and even more in 12.0) to avoid multiple options for the same behavior.

You can use `form_view_initial_mode` in the action context to do what you described.

@sswapnesh thanks for your contribution, it helped me to understand what happened ; I will fix the issue in 11.0 (see mentioned commit).

Regards,

Martin
